ui/src/components/directory/topbar/eclSearch/builder/Refinement.vue

```diff
@@ -313,14 +323,31 @@ async function updateValueTreeRoots(): Promise<void> {
 async function updateIsValidProperty(): Promise<void> {
   if (props.focus?.iri === SNOMED.ANY) isValidProperty.value = true;
   else if (props.focus && hasProperty.value) {
-    const request: FunctionRequest = {
-      functionIri: IM_FUNCTION.ALLOWABLE_PROPERTIES,
-      arguments: [
-        { parameter: "focus", valueObject: props.focus },
-        { parameter: "searchIri", valueData: props.value.property.concept?.iri }
-      ]
-    };
-    isValidProperty.value = await FunctionService.runAskFunction(request);
+    const queryRequest: QueryRequest = {
```

Comment:
No need to recreate the queryRequest, the imQueryForPropertySearch should already be built properly. The only thing needed is the addition of the askIri. (osQueryForPropertySearch is used only for SNOMED.ANY)